rosidl_typesupport_c/test/mocking_utils/patch.hpp

```diff
@@ -0,0 +1,578 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
Hm, we now have half-a-dozen copies of this header file sprinkled around.  Can we put them in one place?

Reply:
I guess I'm the straw that broke the camel's back. I can find a location for it.

Reply:
> I guess I'm the straw that broke the camel's back. I can find a location for it.

That would be great.  Just to be clear, I'm fine putting this PR in for now and then doing a follow-up for that, I just thought we should address it.